Kitodo-Docket/pom.xml

```diff
@@ -70,6 +70,12 @@
             <groupId>org.apache.pdfbox</groupId>
             <artifactId>pdfbox</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>commons-collections</groupId>
+            <artifactId>commons-collections</artifactId>
```

Comment:
- used version of commons-collections is very, very old and should be replaced by commons-collections4
- in the new created unit test there is no usage of this library or I did not see the usage

Reply:
Is then the used scope `test` right? Should then the scope not changed to the default value so this dependency is visible and not hidden solved?

Reply:
It is only needed for the test to work. Don’t ask me why

Reply:
The reason is, that this dependency exists somewhere else too and is resolved there and as the application provide all needed dependencies the dependency  here is solved in a "hidden" (or magic) way.

Reply:
I removed `test` scope